gef-ui/component-tests/partials/about-facility.component-test.js

```diff
@@ -0,0 +1,188 @@
+import { isFacilityEndDateFeatureFlagEnabled } from '@ukef/dtfs2-common';
+
+const pageRenderer = require('../pageRenderer');
+const { FACILITY_TYPE } = require('../../server/constants');
+
+const page = 'partials/about-facility.njk';
+const render = pageRenderer(page);
+
+describe(page, () => {
+  let wrapper;
+
+  const facilityName = 'some facility name';
+  const monthsOfCover = '6';
+  const coverStartDateDay = '1';
+  const coverStartDateMonth = '2';
+  const coverStartDateYear = '1970';
+  const coverEndDateDay = '3';
+  const coverEndDateMonth = '4';
+  const coverEndDateYear = '1971';
+  const dealId = '61e54dd5b578247e14575882';
+  const facilityId = '666862d9140a08222cbd69e7';
+
+  const params = {
+    facilityType: FACILITY_TYPE.CASH,
+    facilityName,
+    hasBeenIssued: false,
+    monthsOfCover,
+    shouldCoverStartOnSubmission: null,
+    coverStartDateDay,
+    coverStartDateMonth,
+    coverStartDateYear,
+    coverEndDateDay,
+    coverEndDateMonth,
+    coverEndDateYear,
+    facilityTypeString: FACILITY_TYPE.CASH.toLowerCase(),
+    dealId,
+    facilityId,
+    status: null,
+  };
+
+  beforeEach(() => {
+    wrapper = render(params);
+  });
+
+  it(`only renders the 'Back to previous page' link if the status does not equal 'change'`, () => {
```

Comment:
Good shout - I can't actually find any mention in the code base of any statuses other than `'change'` (might make sense to for it to be `?change=true` in the URL rather than `?status=change`, but this is again definitely out of scope) so I think I'll leave it as is if you're happy?

Reply:
Yep sounds good